heron/lib/widgets/BookmarksPanel.js

```diff
@@ -645,21 +698,24 @@ Heron.widgets.BookmarksPanel = Ext.extend(Heron.widgets.HTMLPanel, {
 	},
 
 	isValidBookmark: function (context) {
+        var excludeLayers = ['Editor', 'OpenLayers.Handler.Polygon', 'OpenLayers.Handler.RegularPolygon', 'OpenLayers.Handler.Path', 'OpenLayers.Handler.Point']; // Layer-names to be excluded from validation, edit-Layers

```

Comment:
It is assumed here that the name of the Editing Layer is always 'Editor'. This is not always the case. For example in the KadViewer http://kadviewer.kademo.nl it is called 'Tekenlaag'. I think you can derive the name of the Editor Layer by checking map.editor, also when restoring a bookmark.